<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reserver</class>
 <widget class="QDialog" name="Reserver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Reservation</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>601</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>0</y>
       <width>481</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>AJOUTER UNE RESERVATION</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutid">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutdate">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutmontant">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Boutonajouter">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutdate2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutype">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>220</y>
       <width>451</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>MODIFIER UNE RESERVATION</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mod1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="mod2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="mod3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Appliquer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>380</y>
       <width>461</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>SUPPRIMER UNE RESERVATION</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="delete_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>442</x>
       <y>0</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>AFFICHER UNE RESERVATION </string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>530</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>TRI</string>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>TRIER PAR </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NOM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>500</y>
       <width>251</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>RECHERCHE UNE RESERVATION</string>
     </property>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cher1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="che2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="che3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stat</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Client</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>461</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>AJOUTER UN CLIENT</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutid_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutnom_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutprenom_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Boutonajouter_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutadresse_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ajoutel_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Tel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>461</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>MODIFIER UN CLIENT</string>
     </property>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenomm_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="idm_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomm_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Appliquer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>400</y>
       <width>461</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>SUPPRIMER UN CLIENT</string>
     </property>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="delete_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>520</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>RECHERCHER UN CLIENT</string>
     </property>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cher1_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="che2_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="che3_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>601</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>482</x>
       <y>0</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>AFFICHER UN CLIENT</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>statistiques</string>
    </attribute>
    <widget class="QFrame" name="horizontalFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>761</width>
       <height>671</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
